O/ORC/build_tarballs.jl

```diff
@@ -0,0 +1,41 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ORC"
+version = v"4.31.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://gstreamer.freedesktop.org/src/orc/orc-0.4.31.tar.xz", "a0ab5f10a6a9ae7c3a6b4218246564c3bf00d657cbdf587e6d34ec3ef0616075")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd orc-0.4.31
+mkdir build && cd build
+meson .. --cross-file=${MESON_TARGET_TOOLCHAIN}
+ninja
+ninja install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("liborc-test", :liborc_test),
```

Comment:
```suggestion
    LibraryProduct("liborc-test-0.4", :liborc_test),
```

Reply:
Oh got it. I think these statements were generated by the wizard though, so maybe it made a mistake?

Reply:
But you inserted the name

Reply:
Well, I inserted the names with underscores, but not the ones with dashes, I don't think? I pasted the products part of the wizard below

```
[press: d=done, a=all, n=none]
 > [X] bin/orc-bugreport
   [X] bin/orcc
   [X] lib/liborc-0.4.so.0.31.0
   [X] lib/liborc-test-0.4.so.0.31.0
Please provide a unique variable name for each build artifact:
lib/liborc-test-0.4.so.0.31.0:
> liborc_test

bin/orcc (default 'orcc'):
> 

lib/liborc-0.4.so.0.31.0:
> liborc

bin/orc-bugreport:
> orc_bugreport
```

Reply:
Right, sorry, this is a bug in the wizard, which happens when libraries have stupid names, like that.  We've had in another occasion too